bangazonapi/views/product.py

```diff
@@ -6,7 +6,7 @@
 from rest_framework.response import Response
 from rest_framework import serializers
 from rest_framework import status
-from bangazonapi.models import Product, Customer, ProductCategory
+from bangazonapi.models import Product, Customer, ProductCategory, product
```

Comment:
Why are there two `Product` and `product` models?

Reply:
it is not used .. didn't pay attention to that I will delete it and test then push the update.